content/zh-tw/agile-software-development.md

```diff
@@ -0,0 +1,27 @@
+---
+title: 敏捷軟體開發
+status: Completed
+category: 概念
+tags: ["方法論", "", ""]
+---
+
+## 是什麼 {#what-it-is}
+
+敏捷軟體開發是一系列強調迭代開發週期和自我組織團隊的實踐方法。
+與僅在專案結束時才生成價值的瀑布式專案相比，
+敏捷軟體開發著重於價值的持續、增量式的交付，並對過程本身進行漸進式的改善。
+
+## 解決的問題 {#problem-it-addresses}
+
+在軟體專案中，定義、傳達和理解所有利害關係人的需求非常困難，甚至是不可能的。
+然而，客戶希望他們的軟體專案能有高品質，在預算、時程及專案範圍內交付。
+相較於於瀑布式策略，敏捷軟體開發的週期特性使其能持續適應需求，並且能更快地適應其他所有情況。
+
+## 如何幫助 {#how-it-helps}
```

Comment:
```suggestion
## 如何幫助我們 {#how-it-helps}
```
Modified based on discussion in [slack](https://cloud-native.slack.com/archives/C055KERTNCX/p1692440415536599?thread_ts=1692324528.708459&cid=C055KERTNCX)